civictechprojects/static/css/_bootstrapoverride.scss

```diff
@@ -28,9 +28,12 @@ $font-family-sans-serif: 'Montserrat',-apple-system, BlinkMacSystemFont, "Segoe
 
 //add DL colors to themes for button variants, etc
 $theme-colors: (
-  "dl-orange": #f79e02,
-  "dl-orange-darker": #de8e01
+  "primary": #f79e02, // use this for "action"
```

Comment:
Of note here are the primary, success, and danger keys - these are ones Bootstrap [needs to have](https://getbootstrap.com/docs/4.0/getting-started/theming/#required-keys) so we'll want to keep an eye out on buttons, forms, and input elements to make sure they're doing what we expect now that they've been rethemed.